src/test/kotlin/org/partiql/scribe/targets/sandbox/SandboxTargetSuite.kt

```diff
@@ -0,0 +1,14 @@
+package org.partiql.scribe.targets.sandbox
```

Comment:
self-review: including a basic sandbox test suite while we get things working. this allows us to test a single query more easily. we can likely delete this test suite after we get some CLI working again.